HTML/mockup-mogo/js/task6.js

```diff
@@ -0,0 +1,27 @@
+import {loadedElements} from './elements.js';
+
+export function displayAccordion() {
+  loadedElements.then((elements) => {
```

Comment:
FYI: you can use destructuring to pick-up properties from object:
`loadedElements.then(({optionsPart, optionTop, paragraph, arrow, picture}) => {`